src/api/Gemfile

```diff
@@ -119,6 +119,8 @@ group :test do
   gem 'rails-controller-testing'
   # To generate random data
   gem 'rantly', '>= 1.1.0'
+  # To ensure consistent Git commits
+  gem 'git-cop', git: 'https://github.com/Ana06/git-cop.git', branch: 'travis-ci'
```

Comment:
I know, I am going to send a PR to the gem with the changes and then we will use the next release of the gem. But I created the PR to discuss what cops we want to use.